### ISSUE TITLE
MatplotlibDeprecationWarnings in Transformations.plot_timeheight

### ISSUE THREAD

**Issue by martin-rdz** [lacros-tropos/larda#3]
Using matplotlib 3.3.0, two deprication warnings are displayed:

```
../larda/pyLARDA/Transformations.py:607: MatplotlibDeprecationWarning: Passing parameters norm and vmin/
vmax simultaneously is deprecated since 3.3 and will become an error two minor releases later. Please pa
ss vmin/vmax directly to the norm when creating it.
```

```
../larda/pyLARDA/Transformations.py:607: MatplotlibDeprecationWarning: shading='flat' when X and Y have
the same dimensions as C is deprecated since 3.3.  Either specify the corners of the quadrilaterals with
 X and Y, or pass shading='auto', 'nearest' or 'gouraud', or set rcParams['pcolor.shading'].  This will
become an error two minor releases later.
```

**Comment by martin-rdz** [comment]
That should be covered by the new plotting function introduced in 117fff588e49c, or @KarlJohnsonnn?

**Comment by KarlJohnsonnn** [comment]
Fixed with the new commit 16eb5e0ba4c3842e25f8586c9ace59b520312af7.
